packages/musescore.rb

```diff
@@ -0,0 +1,33 @@
+require 'package'
+ 
+class Musescore < Package
+  description "The world's most popular notation app"
+  homepage 'musescore.org'
+  version '4.1.0'
+  license 'GPL-3'
+  compatibility 'aarch64, x86_64'
```

Comment:
Only x86_64 compatible?

Reply:
yes

Reply:
So does this need to be changed to `compatibility 'x86_64'`?